app/javascript/mastodon/features/alt_text_modal/index.tsx

```diff
@@ -105,6 +106,11 @@ const Preview: React.FC<{
   position: FocalPoint;
   onPositionChange: (arg0: FocalPoint) => void;
 }> = ({ mediaId, position, onPositionChange }) => {
+  const [x, y] = position;
+  const [style, api] = useSpring(() => ({
+    left: `${x * 100}%`,
+    top: `${y * 100}%`,
+  }));
```

Comment:
Should we use a different name than `api`? This is pretty non-specific and confusing.

Reply:
ideally we wouldn't use `api` at all. this should provide similar behavior:
```suggestion
  const style = useSpring({
    left: `${x * 100}%`,
    top: `${y * 100}%`,
    immediate: reduceMotion || draggingRef?.current === true,
  });
```
however if we keep this pattern, i think `immediate` should be set to `reduceMotion`?